src/main/java/Storage.java

```diff
@@ -0,0 +1,121 @@
+import java.io.*;
+
+// Level 7: Created Storage class to maintain storage state
+public class Storage {
+
+    private final File storageFile;
```

Comment:
I noticed a similar issue here as in your Duke class - perhaps name your constants using the constant naming convention?